Readme.md

```diff
@@ -1,23 +1,57 @@
+
+## Implementation Details
+
+In this example, [ASPxFileManager](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxFileManager) contains files that can be opened in [ASPxSpreadsheet](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet) or [ASPxRichEdit](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxRichEdit.ASPxRichEdit) component. When a user clicks a file, the [ASPxClientFileManager.SelectedFileChanged](https://docs.devexpress.com/AspNet/js-ASPxClientFileManager.SelectedFileChanged) event fires. The event handler shows a popup and sends callback to server.
+
+```jscript
+function OnSelectedFileChanged(s, e) {
+    if (e.file != null) {
+        PopupWithDocument.Show();
+        PopupWithDocument.PerformCallback(e.file.GetFullName());
+    }
+}
+```
+On the server, the [WindowCallback](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControlBase.WindowCallback) event handler determines a format of the selected file and opens the file in the [ASPxSpreadsheet](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxSpreadsheet.ASPxSpreadsheet) component for [spreadcheet formats](https://docs.devexpress.com/OfficeFileAPI/DevExpress.Spreadsheet.DocumentFormat._members#fields); otherwise the document is opened in the [ASPxRichEdit](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxRichEdit.ASPxRichEdit) component.
+
+```scharp
+protected void PopupWithDocument_WindowCallback(object source, DevExpress.Web.PopupWindowCallbackArgs e) {
+String fullFileName = e.Parameter;
 
-<p>This example demonstrates how to open a file which is selected in ASPxFileManager. The <a href="https://documentation.devexpress.com/#AspNet/DevExpressWebScriptsASPxClientFileManager_SelectedFileChangedtopic">ASPxClientFileManager.SelectedFileChanged</a> event is handled to send a callback to the server, and open the file using ASPxSpreadsheet or ASPxRichEdit inside a popup.</p>
+object format = DocumentFormatHelper.GetFormat(fullFileName);
+if (format == null) return;
 
-<br/>
+Boolean isSpreadsheet = format is DevExpress.Spreadsheet.DocumentFormat;
+ASPxSpreadsheet1.Visible = isSpreadsheet;
+ASPxRichEdit1.Visible = !isSpreadsheet;
+
+var docId = Guid.NewGuid().ToString();
+var docPath =  Server.MapPath(fullFileName);
+
+if (isSpreadsheet)
+  ASPxSpreadsheet1.Open(docId, (DevExpress.Spreadsheet.DocumentFormat)format, () => File.ReadAllBytes(docPath));
+else
+  ASPxRichEdit1.Open(docId, (DevExpress.XtraRichEdit.DocumentFormat)format, () => File.ReadAllBytes(docPath));
+}
+```
+
+## Files to Review
+
+* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx](./VB/Default.aspx))
+* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
 
+## More Examples
 
+* [File Manager for ASP.NET Web Forms - How to implement a custom document management for different document types](https://github.com/DevExpress-Examples/asp-net-web-forms-file-manager-custom-document-management)
```

Comment:
I'd like to update the title of this GitHub readme. Can you update the actual GitHub doc as well?

```suggestion
* [File Manager for ASP.NET Web Forms - How to implement custom document management for different document types](https://github.com/DevExpress-Examples/asp-net-web-forms-file-manager-custom-document-management)
```